src/platform_impl/windows/util.rs

```diff
@@ -149,6 +149,9 @@ pub fn is_minimized(window: HWND) -> bool {
     unsafe { IsIconic(window) != false.into() }
 }
 
+#[allow(clippy::upper_case_acronyms)]
+pub type HINSTANCE = isize;
```

Comment:
It was said that `HMODULE` is the same as `HINSTANCE` nowadays, the difference was only on 16bit windows, so we likely should use `HMODULE`.

https://learn.microsoft.com/en-us/windows/win32/winprog/windows-data-types